clients/gtest/CMakeLists.txt

```diff
@@ -22,9 +22,9 @@ include_directories(${GTEST_INCLUDE_DIRS})
 
 set(hipsolver_test_source
   hipsolver_gtest_main.cpp
-  geqrf_gtest.cpp
-  getrf_gtest.cpp
   getrs_gtest.cpp
+  getrf_gtest.cpp
+  geqrf_gtest.cpp
```

Comment:
What problem did this fix?

Reply:
For some reason getrs has been having memory access faults on CI when using that original test ordering. Juan said we had a similar issue in rocSOLVER before, and he solved it by reordering the tests. Apparently there may be a mystery bug in our client code (e.g. device_vectors).